dashboards/service-level.jsonnet

```diff
@@ -43,13 +43,13 @@ stdlib.dashboard(
   .addTarget(
     grafana.prometheus.target(
       'sum(
-        slo_errors_per_request:ratio_rate1h{pipeline=~"$pipeline", service=~"$service", customer=~"$customer", installation=~"$installation", cluster_id=~"$cluster_id"} > on (service, cluster_id) group_left (cluster_type, class) slo_threshold_high
+        slo_errors_per_request:ratio_rate1h{pipeline=~"$pipeline", service=~"$service", customer=~"$customer", installation=~"$installation", cluster_id=~"$cluster_id"} > on (service, cluster_id, installation, label_application_giantswarm_io_team) group_left (cluster_type, class) slo_threshold_high
```

Comment:
Installation label is added to fix https://github.com/giantswarm/giantswarm/issues/29915
The team label because the dashboards do not work after cilium ownership was changed